docs/example-scenario/data/data-warehouse.md

```diff
@@ -0,0 +1,122 @@
+
+For other availability topics, see the [availability checklist][availability] in the Azure Architecure Center.
+
+### Scalability
+
+The [massively parallel processing architecture](/azure/sql-data-warehouse/massively-parallel-processing-mpp-architecture) of SQL Data Warehouse provides scalability and high performance. [SQL Data Warehouse Gen2](/azure/sql-data-warehouse/memory-and-concurrency-limits) offers the greatest level of scale (up to 30,000 Data Warehouse Units) and providing unlimited columnar storage. You can also [scale SQL Data Warehouse on demand](/azure/sql-data-warehouse/sql-data-warehouse-manage-compute-overview), reducing or even pausing compute during periods of low demand or inactivity to lower your overall costs.
+
+Azure Analysis Services can be [configured for scale-out](/azure/analysis-services/analysis-services-scale-out). With scale-out, client queries can be distributed among multiple query replicas in a query pool, reducing response times during high query workloads. You can also separate processing from the query pool, ensuring client queries are not adversely affected by processing operations. 
+
+For other scalability topics, see the [scalability checklist][scalability] in the Azure Architecure Center.
+
+### Security
+
+The [SQL Data Warehouse security model](/azure/sql-data-warehouse/sql-data-warehouse-overview-manage-security) provides connection security, [authentication and authorization](/azure/sql-data-warehouse/sql-data-warehouse-authentication) via Azure AD or SQL Server authentication, and encryption. [Azure Analysis Services](/azure/analysis-services/analysis-services-manage-users) uses Azure Active Directory (Azure AD) for identity management and user authentication. 
+
+For general guidance on designing secure solutions, see the [Azure Security Documentation][security].
+
+### Resiliency
+
+For general guidance on designing resilient solutions, see [Designing resilient applications for Azure][resiliency].
+
+## Pricing
+
+To explore the cost of running this solution, all of the services are pre-configured in the cost calculator.  To see how the pricing would change for your particular use case, change the appropriate variables to match your expected traffic.
+
+We have provided three sample cost profiles based on amount of traffic you expect to get:
+
+* [Small][small-pricing]: this correlates to a small data pipeline with part-time availability.
+* [Medium][medium-pricing]: this correlates to a midsize data pipeline with full availabilty.
+* [Large][large-pricing]: this correlates to a large data pipeline with full availability.
+
+## Related Resources
+
+This example scenario is based on a version of this architecture used by  [Maritz Motivation Solutions](https://maritz.com) For more information, see their [customer story][source-document]. 
```

Comment:
How about moving this to a new section called "Next steps" or highlighting that if you want to start implementing it this is where you need to go.

Reply:
If we add a new section for this, we should standardize that across other relevant scenarios.